### PR TITLE
[feat] Add db health check

### PR DESCRIPTION
**Pull request by lukaszewczak**
Hi @gergelyke ,

This PR includes directly example to health check postgres database.

Regards,
Łukasz

### CODE REVIEW

**Comment by lukaszewczak** [comment]
If now this example is ok, I will prepare similar examples for other databases.

File: example/postgres/index.js [hunk 1]
```diff
@@ -32,7 +36,8 @@ async function startServer() {
     logger: console.log,
     signal: 'SIGINT',
     healthChecks: {
-      '/healthcheck': onHealthCheck
+      '/healthcheck': onHealthCheck,
```
Comment (gergelyke) [inline example/postgres/index.js]: shouldn't `onHealthcheck` contain the check for the db? if the db isn't healthy, most probably the app won't be neither - what do you think?

Comment (lukaszewczak) [reply example/postgres/index.js]: Yes, you are right, I change this.

**Comment by gergelyke** [comment]
thanks a lot @lukaszewczak - looks great!

**Pull request merged**